TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.js

```diff
@@ -311,6 +320,27 @@ function (Ember, $, FileUpload, template, cropTemplate, BooleanEditor, Spinner,
 						that.fileUploaded(result);
 						that._hideImageLoader();
 					});
+
+					if (previewImageUrlBeforeLoading !== that.get('_previewImageUri')) {
+						// We overwrite the value so the inspector will enable the 'Apply' button so the reloadAfterChange
+						// will trigger as the image is visually changed even though the actual value (variant identifier)
+						// is still the same.
+						that.set('value', value);
+					}
+				},
+				onLoad: function(event) {
+					if (this._assetIdentifier) {
```

Comment:
yes, couldn't find a way around that. It's too many reloads of the image, but I can not find a way to just listen to POST requests.